data/json/items/armor.json

```diff
@@ -226,6 +226,22 @@
     "environmental_protection": 2,
     "flags": [ "STURDY", "BELTED", "BLOCK_WHILE_WORN", "WATER_FRIENDLY" ]
   },
+  {
+    "id": "armguard_acidchitin",
+    "copy-from": "armguard_chitin",
+    "type": "ARMOR",
+    "name": "pair of biosilicified chitin arm guards",
+    "name_plural": "pairs of biosilicified chitin arm guards",
+    "description": "A pair of arm guards crafted from carefully cleaned and pruned biosilicified exoskeletons of acidic ants.  Acid-resistant and very durable.",
+    "weight": 441,
+    "volume": 11,
+    "price": 25000,
+    "material": [ "acidchitin" ],
+    "encumbrance": 15,
+    "warmth": 15,
+    "material_thickness": 4,
+    "environmental_protection": 2
```

Comment:
Did you test your own PR? Debug message states this assignment doesn't update the value (value of environmental protection of  "armguard_chitin" is also 2, so no need to write the same here).
Please run the game and clean up everything causing these debug messages.

Also, you can use special fields "relative" and "proportional" like this:
```
    "relative": { "volume": 1, "warmth": 5, "material_thickness": 1 },
    "proportional": { "weight": 1.125, "price": 1.25, "encumbrance": 1.5 }
```
Technically, it will give the same results, but in case of review of regular chitin armor, the biosilicified version will be rebalanced automatically.

Reply:
I won't have time to look at this more fully for a few days, but instead of waiting even more, where is this debug message stating this so I can review it? I was not aware of such a thing.

Reply:
It appears on the screen right after loading the game with your changes. Just run the game and you will have no choice to avoid them.

Reply:
Ok, I realize what you mean, the errors that prompt. I wasn't getting these errors because I had all these changes confined within a mod, but once I overwrote the core files instead, I got the errors you were referring to.

Also, I'd recommend adding the relative and proportional to JSON doc files.